org-code-javabuilder/playground/src/main/java/org/code/playground/Board.java

```diff
@@ -141,7 +141,14 @@ public void removeItem(Item item) {
    * @param filename the name of the sound file from the asset manager to play
    * @throws FileNotFoundException when the sound file cannot be found.
    */
-  public void playSound(String filename) throws FileNotFoundException {}
+  public void playSound(String filename) throws FileNotFoundException {
+    HashMap<String, String> details = new HashMap<>();
+    details.put("filename", filename);
```

Comment:
Sanchit added this new class with constants for sending messages which already has a FILENAME constant: https://github.com/code-dot-org/javabuilder/blob/main/org-code-javabuilder/protocol/src/main/java/org/code/protocol/ClientMessageDetailKeys.java